src/renderers/common/Renderer.js

```diff
@@ -1121,14 +1121,21 @@ class Renderer {
 		const { width, height } = this.getDrawingBufferSize( _drawingBufferSize );
 		const { depth, stencil } = this;
 
+		// to improve performance we only want to use HalfFloatType if necessary
+		// HalfFloatType is required when a) tone mapping is used or b) the backend specifically requests it
+
+		const type = ( useToneMapping || this.backend.needsHalfFloatFrameBufferTarget() ) ? HalfFloatType : UnsignedByteType;
```

Comment:
Side note: When using `UnsignedByteType`, it is in general recommended to use a sRGB storage format for the framebuffer for better storage precision (see https://stackoverflow.com/a/47098507/5250847). However, these framebuffer formats are buggy in WebGL and do not consistently work across all platforms.